### ISSUE TITLE
Add option --max-geometry

### ISSUE THREAD

**Issue by fedov** [derf/feh#327]
This is a feature request. Add an option --max-geometry [width x height].
In combination with a non-tiling window manager feh opens images in a window with the same dimensions as the image, even if the window doesn't fit the screen. This is almost never what I want.
The proposed option would make sure, that windows never (initially) exceed a certain size. Images smaller than max-geometry should not be affected by this option.
Like --geometry it should not enforce the geometry.

**Comment by ulteq** [comment]
> In combination with a non-tiling window manager feh opens images in a window with the same dimensions as the image, even if the window doesn't fit the screen.

That's not how it is supposed to be, is it?

The manual says:

> `--no-screen-clip`
> By default, window sizes are limited to the screen size.  With this option, windows will have the size of the image inside them.

**Comment by derf** [comment]
Indeed. feh should not create images larger than the screen by default. Which window manager are you using? How are you calling feh? What's the content of your ~/.config/feh/themes?

**Comment by ulteq** [comment]
@fedov, could you try v2.27 and see if this is still an issue for you?
